Source/WebInspectorUI/UserInterface/Views/LogContentView.js

```diff
@@ -89,6 +89,14 @@ WI.LogContentView = class LogContentView extends WI.ContentView
             this._emulateUserGestureNavigationItemGroup = new WI.GroupNavigationItem([this._emulateUserGestureNavigationItem, new WI.DividerNavigationItem]);
         }
 
+        // 'Show Timestamp' toggle
+        this._showTimestampsNavigationItem = new WI.CheckboxNavigationItem("show-timestamps", WI.UIString("Show Timestamps", "Show Timestamps @ Console", "Checkbox shown in the Console to show timestamps."), WI.settings.showTimestamps.value);
+        this._showTimestampsNavigationItem.tooltip = WI.UIString("Show timestamp for each console log");
```

Comment:
It seems like we generally use "console log" to refer to the messages as a whole, so maybe "console message" would be more accurate here?

Reply:
does this really even need a tooltip?  it's already showing pretty self-explanatory text

definitely agree with @patrickangle about the "console message" part tho if we do decide to keep a tooltip :)